doc/snipe.txt

```diff
@@ -67,6 +67,15 @@ Dvorak users, e.g.
 >
     let g:snipe_jump_tokens = 'AOEUIDHTNS'
 <
+The jump tokens are highlighted in a configurable color. The
+default colors are #fabd2f for gui mode and 7 (light grey). To
+override these values, specify the the variables (for gui mode and
+terminal mode respectively):
+>
+    let g:snipe_highlight_gui_color = '#dfffaf'
+
+    let g:snipe_highlight_cterm_color = 'red'
+<
```

Comment:
So I actually have no clue how this works, just cargo-culted it from easymotion. If you know how it works, could you say a few words?

Reply:
So in `highlight.vim` if the global variable of the above naming convention exists, it uses it, else it defaults to the values you originally specified.

Reply:
I can follow the code flow, just not sure, e.g. what the difference is between cterm and cterm256. Also, what the `InitializeHLGroup` function is really doing.

Reply:
So as the code is written, it doesn't even matter having the cterm vs cterm256 and that conditional and the redundant dictionary values can be deleted.  It checks if the terminal is running in 256 color mode and sets the value for the overridden 256 value (or default), otherwise, it sets just the cterm value (or default).  Right now they are the same but a user could set both variables if they sometimes use their `vimrc` on a terminal without 256 color support (valid values are 0-15) or on a 256 color terminal and have a different default for 256 color terminals.  Haha, after typing this, maybe you should just keep it this way to leave that flexibility that I described above.

Reply:
OK I understand. Should the default for cterm256 be something different? I get what the string interpolation is, I guess the place to look would be `:h cterm` and `h: highlight`

Reply:
So yes, the default for cterm256 could be 16-255, yes.